meshtastic/__main__.py

```diff
@@ -1794,12 +1810,40 @@ def addRemoteAdminArgs(parser: argparse.ArgumentParser) -> argparse.ArgumentPars
         action="store_true",
     )
 
+    group.add_argument(
+        "--backup-prefs",
+        help="Tell the destination node to create a backup preferences file."
+        "Location: 0 for local flash, 1 for SD card.",
+        default=admin_pb2.AdminMessage.BackupLocation.FLASH,
+        nargs="?",
+        const=0,
```

Comment:
You probably want `default` to stay `None` (that's what's passed if the argument isn't passed on the command line at all) and to set the `admin_pb2.<...>` value as `const` (which is what's passed if the argument is passed bare, without a real value).